tests/sparseml/pytorch/utils/test_logger.py

```diff
@@ -46,7 +45,7 @@
             or True
         ),
         *([WANDBLogger()] if WANDBLogger.available() else []),
-        *([ClearMLLogger()] if ClearMLLogger.available() else []),
+        # *([ClearMLLogger()] if ClearMLLogger.available() else []),
```

Comment:
Definitely do not want to stall landing this PR, however, let's clear the ClearML issues once and for all, it has been slowing us back significantly.

Reply:
So in the context of this PR I'd keep line 48 commented out, but leave a TODO comment so that there is a footprint indicating that we should fix it.